snmp/datadog_checks/snmp/snmp.py

```diff
@@ -311,6 +313,18 @@ def fetch_sysobject_oid(self, config):
         self.log.debug('Returned vars: %s', var_binds)
         return var_binds[0][1].prettyPrint()
 
+    def _profile_for_sysobject_oid(self, sys_object_oid):
+        """Return, if any, a matching profile for sys_object_oid.
+
+        If several profiles match, it will return the longer match, ie the
+        closest one to the sys_object_oid.
+        """
+        oids = [oid for oid in self.profiles_by_oid if fnmatch.fnmatch(sys_object_oid, oid)]
```

Comment:
Could you document the feature in `example.conf.yaml` ? Thx

Reply:
There'll be an refactor just after, I'll do it then if you don't mind (before release).